### ISSUE TITLE
Failures due to yarn --frozen-lockfile

### ISSUE THREAD

**Issue by swcurran** [hyperledger/aries-acapy-docs#4]
The tests for a dependabot PR are failing with the following error:

```
warning @docusaurus/core > @svgr/webpack > @svgr/plugin-svgo > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
warning @docusaurus/core > @docusaurus/mdx-loader > @mdx-js/mdx > remark-parse > trim@0.0.1: Use String.prototype.trim() instead
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
```

Seems to also be happening with the deploy github actions.

Should that yarn parameter be used?

**Comment by amanji** [comment]
I think I need to lock the node version down in the container

**Comment by swcurran** [comment]
(Hopefully) FIxed by #6
